app/controllers/profile_controller.rb

```diff
@@ -22,21 +22,9 @@ def edit
   def update
     @profile = User.find(params[:id])
     @profile.update_attributes!(user_params)
-    if params[:transfer]
-      @profile.update_attribute(:transfer, true)
-    else
-      @profile.update_attribute(:transfer, false)
-    end
-    if params[:graduate]
-      @profile.update_attribute(:graduate, true)
-    else
-      @profile.update_attribute(:graduate, false)
-    end
-    if params[:international]
-      @profile.update_attribute(:international, true)
-    else
-      @profile.update_attribute(:international, false)
-    end
+    @profile.update_attribute(:transfer, params[:transfer] ? true : false)
+    @profile.update_attribute(:transfer, params[:graduate] ? true : false)
```

Comment:
Need to change the attribute that is being updated :transfer -> :graduate, also the next line

Reply:
Good catch!! Thanks!